net.adamec.lib.common.dmn.engine/engine/definition/DmnVariableDefinition.cs

```diff
@@ -140,10 +140,10 @@ public static string NormalizeVariableName(string name)
             if (string.IsNullOrWhiteSpace(retVal))
                 throw new ArgumentException($"Variable name is null or empty",nameof(name));
 
-            if (retVal.Any(c => !(c >= 'a' && c <= 'z' || c >= 'A' && c <= 'Z' || c >= '0' && c <= '9' || c == '_')))
+            if (retVal.Any(c => char.IsLetter(c) || char.IsDigit(c)  || c == '_')))
                 throw new ArgumentException($"Variable name '{name}' contains invalid character", nameof(name));
 
-            if (!(retVal[0] >= 'a' && retVal[0] <= 'z' || retVal[0] >= 'A' && retVal[0] <= 'Z'))
+            if (!char.IsLetter(retVal[0])
```

Comment:
There is a missing closing bracket, so the line should be like this
```
if (!char.IsLetter(retVal[0]))
```

Reply:
and here as well, corrected the statement